src/inputs/input/input.jsx

```diff
@@ -29,46 +50,133 @@ const propTypes = {
      * Deprecated prop. Please use `disable` instead.
      */
     disabled: PropTypes.bool,
+    /**
+     * Indicates that the input has an error.
+     */
     error: PropTypes.oneOfType([
         PropTypes.bool,
         PropTypes.string,
     ]),
+    /**
+     * An input can take on the size of its container.
+     */
     fluid: PropTypes.bool,
+    /**
+     * Tells the component whether to be in guide or no guide mode.
+     */
     guide: PropTypes.bool,
+    /**
+     * Optional icon to display inside the input.
+     */
     icon: PropTypes.oneOfType([
         PropTypes.shape({}),
         PropTypes.string,
     ]),
+    /**
+     * The id is used for the a label\'s for.
+     */
     id: PropTypes.string,
+    /**
+     * Format to appear on dark backgrounds.
+     */
     inverse: PropTypes.bool,
+    /**
+     * Changes the general behavior of the Text Mask component.
```

Comment:
Maybe some more information here would be helpful, but I'd have to read up on what this does before making any recommendations.  NAB for now.

Reply:
Please check this commit https://github.com/saddlebackdev/react-cm-ui/pull/353/commits/8429b8ffd987325a6ae504db3aa2341b60de60ef

Reply:
Looks like no changes to Line 84.  **_Not a Blocker_**